Documentation/proposals/readme.md

```diff
@@ -0,0 +1,18 @@
+# Prometheus-Operator proposals
+
+This section is used to document proposals that the prom-operator team feels go beyond the scope of a simple GitHub issue.
+
+## Quickstart
+1. Fork this repository and clone your forked repository
+2. Run `new.sh` from this directory with a new proposal number and title as arguments:
+   ```bash
+   ./new.sh 0005 awesome-new-feature
+   ```
+3. Fill out the information block at the top of the readme, with 'reserved' as the status.
+4. Commit/push and make a pull request to this repository
+5. Create a PR merging your forked repo (with the branch) into `prometheus-operator:main`
+   - Set the title of MR to "Reserving Proposal 0001" (replacing 0001 with your proposal number)
```

Comment:
```suggestion
   - Set the title of the pull request to "Reserving Proposal 0001" (replacing 0001 with your proposal number)
```

Reply:
You caught me, (I spend most of my day using GitLab...)